dynawo/sources/Models/Modelica/Dynawo/Electrical/HVDC/HvdcVSC/BaseControls/ActivePowerControlSide.mo

```diff
@@ -13,6 +13,7 @@ within Dynawo.Electrical.HVDC.HvdcVSC.BaseControls;
 */
 
 model ActivePowerControlSide "Active Power Control Side of the HVDC link"
+  import Dynawo;
```

Comment:
Not necessary

Reply:
Error after removing it : 

[Dynawo.Electrical.HVDC.HvdcVSC.BaseControls.ActivePowerControlSide: 65:3-66:115]: [Class Dynawo.Electrical.HVDC.HvdcVSC.BaseControls.ACVoltageControl.ACVoltageControl not found in scope ActivePowerControlSide.]